.github/workflows/authors.yml

```diff
@@ -0,0 +1,21 @@
+name: "authors update"
```

Comment:
```suggestion
name: Update AUTHORS
```

Reply:
I disagree with this, since it's following a similar pattern to https://github.com/nodejs/node/actions/workflows/license-builder.yml which uses `<name> update`.

Reply:
Okay! But maybe the (useless) quotes could be removed?